src/applications/gi-sandbox/containers/SearchPage.jsx

```diff
@@ -53,6 +70,24 @@ export function SearchPage({
               </va-alert>
             </div>
           )}
+          {!error && !smallScreen && tabbedResults[tab]}
+          {smallScreen && (
+            <div>
+              <va-accordion>
+                <va-accordion-item header="Search by name">
+                  <NameSearchForm />
```

Comment:
```suggestion
                  <NameSearchForm />
                  <NameSearchResults />
```

Reply:
Then add `@include media-maxwidth($small-screen)` to css classes as needed inside of `NameSearchResults`